app/web/features/profile/locales/en.json

```
@@ -127,37 +128,61 @@
   "report_reason": "Reason",
   "report_user": "Report this user",
   "leave_reference": {
+    "add_a_reference": "Add a reference",
     "appropriate_behavior": "Appropriate Behavior",
-    "appropriate_explanation": "Awesome –– We hope you had a great time! To help keep our community safe, we want to ask about your interaction with your fellow Coucher.",
+    "appropriate_explanation": "References keep users accountable and reward great experiences. Thank you for doing your part to help keep the community safe, trustworthy, and welcoming for everyone! ",
     "appropriate_question": "Did you feel safe with this person's behavior?",
+    "by_writing_thoughtful": "By writing a thoughtful reference, you are doing your part to keep the community a safe, trustworthy, and welcoming place for everyone!",
```

We need to include the new link `howToLeaveGoodReferenceUrl` to the help page I (probably incorrectly) defined in `routes.ts` below. The help page includes a guide on how to write a helpful reference. I'm not sure how to do this properly, I just used <1></1> lol! I also updated the text a bit after some long chats with gpt:

```suggestion
    "by_writing_thoughtful": "Not sure what to write? Read our guide <1>here</1>. By writing honestly and in detail about your experience with this person, you will directly help other members and contribute to safety and trust in the Couchers community.",
```